pkg/serverless/trace/inferredspan/span_enrichment_test.go

```diff
@@ -207,3 +207,7 @@ func mockInferredSpan() InferredSpan {
 	inferredSpan.Span.SpanID = uint64(8048964810003407541)
 	return inferredSpan
 }
+
+func TestCalculateStartTime(t *testing.T) {
+	assert.Equal(t, int64(1651863561696000000), calculateStartTime(1651863561696))
```

Comment:
Does this work with the assertion first? Also, aren't the enrichment functions affected by this change as well? Should probably change those as well since I used the wrong values on the tests.

Reply:
✅